integration/test/performance_tracker/performance_metrics_test.go

```diff
@@ -0,0 +1,46 @@
+//go:build linux && integration
+// +build linux,integration
+
+package data_collector
```

Comment:
I missed this originally:
1. The package name shouldn't have underscores: https://go.dev/blog/package-names
2. The package name makes sense to follow the directory name: https://go.dev/doc/effective_go#package-names
3. This is not something that _needs_ to be changed now, but I think it would make sense to change the directory name to `./integration/test/performance_test/`, and would be easier to change it sooner rather than later. 
```suggestion
package performancetest
```